### PR TITLE
update rollup config flags to match optimism's recommendations

### PR DESCRIPTION
**Pull request by roberto-bayardo**

### CODE REVIEW

File: geth-entrypoint [hunk 1]
```diff
@@ -68,5 +56,6 @@ exec ./geth \
 	--nat=extip:$HOST_IP \
 	--networkid="$CHAIN_ID" \
```
Comment (mdehoog) [inline geth-entrypoint:57]: we should probably get rid of this?

Comment (roberto-bayardo) [reply geth-entrypoint]: yup done.

File: geth-entrypoint [hunk 2]
```diff
@@ -66,7 +53,7 @@ exec ./geth \
 	--nodiscover \
```
Comment (kaliubuntu0206) [inline geth-entrypoint:53]: Could we remove the `--gcmode=archive` flag now?

**Comment by kaliubuntu0206** [comment]
Also, I would like to recommend exposing `--state.scheme` option via an environment variable https://github.com/base-org/node/issues/132#issuecomment-1784439898

**Comment by roberto-bayardo** [comment]
> Also, I would like to recommend exposing `--state.scheme` option via an environment variable [#132 (comment)](https://github.com/base-org/node/issues/132#issuecomment-1784439898)

this is a good idea but will address it separately.

**Review by mdehoog — APPROVED** [review]

File: geth-entrypoint (hunk 2 above)
Comment (mdehoog) [reply geth-entrypoint:54]: Done in #146

**Pull request merged**